bindings/python/setup.py

```diff
@@ -80,7 +94,7 @@ def target(arch):
         "Intended Audience :: Developers",
         "Intended Audience :: Science/Research",
         "Intended Audience :: Other Audience",
-        "License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)",
+        "License :: OSI Approved :: Apache License, Version 2.0",
```

Comment:
and here. In accordance with [OSI](https://opensource.org/licenses#:~:text=Apache%20License%2C%20Version%202.0)